debian/gen-flat-repo-fixtures.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+set -eux
+
+yes | apt-get install dpkg-dev dpkg-sig apt-utils
+
+SRCDIR=$(readlink -f "$(dirname "$0")")
+OUTPUTDIR=$(realpath "${SRCDIR}/../$1")
+
+mkdir "${OUTPUTDIR}"
+
+cd "$OUTPUTDIR"
+(
+  for CTL in "${SRCDIR}"/asgard/*.ctl
+  do
+    equivs-build --arch ppc64 "${CTL}"
+  done
+)
+
+curl -L https://github.com/pulp/pulp-fixtures/raw/master/common/GPG-PRIVATE-KEY-pulp-qe | gpg --import
+echo "6EDF301256480B9B801EBA3D05A5E6DA269D9D98:6:" | gpg --import-ownertrust
+
+dpkg-sig --sign builder ./*.deb
```

Comment:
This signs the packages. Right? We only do signed Metadata in the other debian fixtures. And i doubt the plugin needs this extra time consuming step.

Reply:
Done. Thanks :)